package.json

```diff
@@ -38,6 +38,9 @@
     "@types/vinyl": "^1.1.29",
     "@types/vinyl-fs": "0.0.28",
     "@types/yeoman-generator": "0.0.30",
+    "bower": "^1.8.0",
```

Comment:
Made it a `~`, presuming we might catch a useful patch in the CI. I'll look into repeatable installs very soon.